### ISSUE TITLE
[Chat Trigger] Remove the 'New Chat' page altogether?

### ISSUE THREAD

**Issue by ear-dev** [WideChat/Rocket.Chat#1176]
Currently after a chat is closed, and the visitor closes the widget with the 'X', the next time the visitor clicks the widget icon they are presented with the 'New Chat' page, which offers a button which will trigger a new session. 

Is it possible to remove this intermediary page and have the widget icon click be the trigger again to start a new chat?

**Comment by Shailesh351** [comment]
@ear-dev 
- For the web, it's a straightforward and minor change. On clicking X we minimize and not route to `New Chat Screen` So it will automatically trigger a new chat when the user clicks on it. However, this also leads to a multiple tabs issue where the screen becomes blank. We will need to handle it. 
- For mobile, there is an open question for design. We are currently not minimizing chat when the user clicks on X, we show 'New Chat Page'. So if we remove this page from the flow. It will start the new chat immediately which is not good UX.  If the user is satisfied with the support and wants to close the chat USING X, it will again and again keep starting the new chat.

**Comment by ear-dev** [comment]
related to story https://github.com/WideChat/Rocket.Chat/issues/1159

**Comment by ear-dev** [comment]
@bhardwajaditya @Shailesh351 On mobile, when you close a chat and you are looking at the 'New Chat' page, if you click one of the back arrows (there are two, system level and app level) you go back to the myViasat "Chat with us." page.

Is it possible to navigate to that page upon clicking the 'X', instead of our 'New Chat' page?

**Comment by bhardwajaditya** [comment]
@ear-dev I don't think we will be able to control in app behaviour for myviasat, we can minimise the widget but going back from seems to be difficult

**Comment by ear-dev** [comment]
What does it look like if we minimize the widget in mobile?  What is then visible?

**Comment by bhardwajaditya** [comment]
@ear-dev I am not sure about it but if I remember from one of the demo, it's a white screen with minimised widget. 

@Shailesh351 can confirm this point

**Comment by Shailesh351** [comment]
@ear-dev @bhardwajaditya It behaves the same as the web. It will minimize chat showing the chat icon. 

It is possible to go back when user clicks X, but that will require changes at MyViasat side. And by doing that we are just moving more and more control away from our Widget side, we should avoid that if possible.

**Comment by ear-dev** [comment]
@bhardwajaditya please remove the New Chat page ONLY in web.....  use the isMobile flag to keep it for now in mobile.  We will design a better solution later.

**Comment by ear-dev** [comment]
https://github.com/WideChat/Rocket.Chat.Livechat/pull/147
